doc/man1/openssl-mac.pod.in

```diff
@@ -48,7 +48,7 @@ Output the MAC in binary form. Uses hexadecimal text format if not specified.
 =item B<-cipher> I<name>
 
 Used by CMAC and GMAC to specify the cipher algorithm.
-For CMAC it must be one of AES-128-CBC, AES-192-CBC, AES-256-CBC or
+For CMAC it must be one of AES-128-CBC, AES-192-CBC, AES-256-CBC, SM4-CBC or
```

Comment:
This might be better worded along the lines of:

> For CMAC it must be a CBC mode cipher, for example AES-128-CBC or SM4-CBC.

CMAC doesn't actually restrict the ciphers to those in this list, just to being CBC mode.

Reply:
Thanks, it makes sense.